src/sentry/integrations/slack/message_builder/issues.py

```diff
@@ -316,31 +317,28 @@ def get_suggested_assignees(
     return []
 
 
-def get_suspect_commit_text(
-    project: Project, event: GroupEvent, commits: Sequence[Mapping[str, Any]] | None = None
-) -> SlackBlock:
+def get_suspect_commit_text(group: Group) -> str | None:
```

Comment:
i would like to see how often this is None, but i don't think there's an easy way to determine if it's None because there's no suspect commit found or because the suspect commit is found after the notification has been sent